Add 403 return errors to URIs dealing with revisions

d00rman opened the pull request:
Forgot to address that in #180 , so adding it separately.

gwicke commented on specs/mediawiki/v1/content.yaml:
```diff
@@ -558,6 +582,28 @@ definitions:
         type: string
       uri:
         type: string
+  accessDenied:
```
This is a https://tools.ietf.org/html/draft-nottingham-http-problem

gwicke commented on specs/mediawiki/v1/content.yaml:
```diff
@@ -558,6 +582,28 @@ definitions:
         type: string
+  accessDenied:
+    required:
+      - code
```
The code is in the http status.

gwicke commented on specs/mediawiki/v1/content.yaml:
```diff
@@ -558,6 +582,28 @@ definitions:
+  accessDenied:
+    required:
+      - code
+      - method
```
I believe this is in a req sub-object.

gwicke commented on specs/mediawiki/v1/content.yaml:
```diff
@@ -558,6 +582,28 @@ definitions:
+        type: string
+      type:
+        type: string
+      description:
```
s/description/detail/

gwicke commented on specs/mediawiki/v1/content.yaml:
```diff
@@ -558,6 +582,28 @@ definitions:
+        type: string
+      description:
+        type: string
+      uri:
```
req.uri?

d00rman replied:
ups, c/p error :)

d00rman commented:
@gwicke I have changed `accessDenied` to `httpError` (and updated the fields accordingly). Should we make it as the default schema for all errors, since we have field whitelisting allowing only certain error properties to see the light of day?

gwicke commented on specs/mediawiki/v1/content.yaml:
```diff
@@ -558,6 +582,21 @@ definitions:
         type: string
+  httpError:
+    required:
+      - method
```
I wonder if we should move 'method' down into a req sub-object that has just the uri & method.
